app/javascript/controllers/text_extraction_controller.js

```diff
@@ -0,0 +1,91 @@
+import { Controller } from '@hotwired/stimulus'
+
+export default class extends Controller {
+  static targets = ['textExtractionLanguages', 'textExtractionDropdown',
+    'selectedLanguages', 'languageWarning', 'dropdownContent']
+
+  static values = { languages: Array }
+
+  connect () {
+    this.element.querySelector('form').reset()
```

Comment:
Can you drop a comment about why the form needs to be reset when the component initializes?

Reply:
Yeah, if you refresh the page and you previously clicked on checkbox or radio buttons they will remained selected which I assume isn't something you would want.

Reply:
I mean, can you add a comment into the js file about this.